src/task 31/main.js

```diff
@@ -0,0 +1,44 @@
+//https://jsonplaceholder.typicode.com/users
+//https://eloquentjavascript.net/author
+var requestURL = "https://eloquentjavascript.net/author"
+var requestTypes = ["text/plain",
+                      "text/html",
+                      "application/json",
+                      "application/rainbows+unicorns"];
+
+function sendRequest(method, url, body = null){ 
```

Comment:
Keep in mind, default argument values were introduced in ES6 specification. Also, if you're neither passing any actual argument nor using it in function - better to remove it from declaration as well.